content/applications/finance/accounting/fiscal_localizations/localizations/germany.rst

```diff
@@ -31,7 +31,134 @@ Export from Odoo to Datev
 feature, the german accounting localization needs to be installed on your Odoo Enterprise database.
 Then you can go in :menuselection:`Accounting --> Reporting --> General Ledger` then click on the
-**Export Datev (csv)** button. 
+**Export Datev (csv)** button.
+
+.. _germany/pos:
+
+Point of Sale in Germany: Technical Security System
+===================================================
+
+Since the new regulations "The Act on Protection against Manipulation of Digital Records"
+(Kassensicherungsverordnung), a **Technical Security System** (also called **TSS** or **TSE**) needs
+to be used for each :doc:`point of sale </applications/sales/point_of_sale>`.
+
+Odoo offers a service that is compliant with the help of `fiskaly <https://fiskaly.com>`_, a
+*cloud-based solution*.
+
+.. important::
+   Since this solution is cloud-based, a working internet connection is required.
+
+.. note::
+   The only VAT rates allowed are given by fiskaly. You can check these rates by consulting:
+   `fiskaly DSFinV-K API: VAT Definition
+   <https://developer.fiskaly.com/api/dsfinvk/v0/#tag/VAT-Definition>`_.
+
+Configuration
+-------------
+
+Modules installation
+~~~~~~~~~~~~~~~~~~~~
+
+#. If your database was created before June 2021, :ref:`upgrade <general/upgrade>` your **Point of
+   Sale** app (`point_of_sale`) and the **Restaurant** module (`pos_restaurant`).
+#. :ref:`Install <general/install>` the **Germany - Certification for Point of Sale**
+   (`l10n_de_pos_cert`) and **Germany - Certification for Point of Sale of type restaurant**
+   (`l10n_de_pos_res_cert`) modules.
+
+   .. tip::
+      If these modules are not listed, :ref:`update the app list <general/install>`.
+
+.. image:: germany/pos-upgrade.png
+   :align: center
+   :alt: Upgrading Odoo Point of Sale from the Apps dashboard
+
+Register your company at the financial authority
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To register your company, go to :menuselection:`Settings --> General Settings --> Companies -->
+Update Info` and fill out the following fields:
+
+- **Company name**
+- Valid **address**
+- **VAT** number
+- **St.-Nr** (Steuernummer): this number is assigned by the tax office to every taxable natural or
+  legal person. (e.g., `2893081508152`)
+- **W-IdNr** (Wirtschafts-Identifikationsnummer): this number is used as a permanent
+  identification number for economically active persons.
+
+You can then **register your company through fiskaly** by opening the *fiskaly* tab and clicking on
+the *fiskaly Registration* button.
+
+.. image:: germany/fiskaly-registration.png
+   :align: center
+   :alt: Button to register a company through fiskaly in Odoo
+
+Once the registration has been finalized, new fields appear:
+
+- **fiskaly organization ID** refers to the ID of your company at the fiskaly side.
+- **fiskaly API key** and **secret** are the credentials the system uses to access the services
+  offered by fiskaly with your company.
```

Comment:
Not sure about _with_ your company
Maybe _for_ or _from_, but I'm not sure of the initial meaning. Or maybe with your company _details_ maybe?